src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,88 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTORY = 0.75f;
+    private int capacity = DEFAULT_CAPACITY;
+    private int size;
+    private Node[] table;
```

Comment:
используй параметризацию
```suggestion
 private Node<K, V>[] table;
```